orm/src/orm.go

```diff
@@ -49,6 +49,18 @@ func FindChallenge(id string) (model.Challenge, error) {
 	return challenge, err
 }
 
+// UpdateChallenge receives an existing Challenge object and updates its value on the database.
+func UpdateChallenge(challenge model.Challenge) error {
+	db := dbConnect()
```

Comment:
Does dbConnect retain connections, or are you opening and closing connections every time?

Reply:
The connections are retained.
Everything is done inside a once.Do on dbConnect:

```bash
func dbConnect() *gorm.DB {
	once.Do(func() {
		[...]
		var err error
		if db, err = gorm.Open(postgres.Open(dsn), &gorm.Config{}); err != nil {
			panic(err)
		}
	[...]
	})
	return db
}
```